modules/imaging_browser/php/viewsession.class.inc

```diff
@@ -221,11 +221,11 @@ class ViewSession extends \NDB_Form
             $SeriesNumber        = $FileObj->getParameter('series_number');
             $SeriesUID           = $FileObj->getParameter('series_instance_uid');
             $EchoTime            = number_format(
-                (int)$FileObj->getParameter('echo_time')*1000,
+                (float)$FileObj->getParameter('echo_time') * 1000,
```

Comment:
This looks really familiar.. didn't we already do this?

Reply:
the PR was sent to bug fix but got closed when bugfix had to be removed so I had to rebase it and send another PR so that it actually makes it to the code base ;)